test/evaluation/inputDescriptorFilterEvaluationHandler.spec.ts

```diff
@@ -127,13 +167,20 @@ describe('inputDescriptorFilterEvaluationHandler tests', () => {
     message2.payload = { 'result': { 'path': ['$', 'issuer'], 'value': 'did:foo:123' }, 'valid': true };
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const evaluationHandler: EvaluationHandler = new InputDescriptorFilterEvaluationHandler(evaluationClient);
-    evaluationHandler.handle(presentationDefinition, new VP(presentation));
+    evaluationHandler.handle(presentationDefinition, presentation);
     expect(evaluationClient.results).toEqual([message, message1, message2]);
   });
 
   it(`input descriptor's constraint.fields.filter match and nested_path`, () => {
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_nested_submission.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.